Eevee.Sleep.Bot/Utils/DiscordMessageMaker/DiscordMessageMakerForRoleRestrict.cs

```diff
@@ -0,0 +1,69 @@
+using Discord;
+using Eevee.Sleep.Bot.Enums;
+using Eevee.Sleep.Bot.Extensions;
+
+namespace Eevee.Sleep.Bot.Utils.DiscordMessageMaker;
+
+public static class DiscordMessageMakerForRoleRestrict {
+    public static Embed MakeRestrictedRoleRemovedMessage(
+        string roleName,
+        IUser user,
+        uint accountAgeDays
+    ) {
+        var serverlink = $"https://discord.com/channels/{ConfigHelper.GetDiscordWorkingGuild()}";
+        var dueTo = user.CreatedAt.ToUniversalTime().AddDays(accountAgeDays).ToString("yyyy-MM-dd HH:mm:ss");
+        string[] messages = [
+            $"The role {roleName} in {serverlink} requires account age of at least {accountAgeDays} days to prevent.",
+            $"You will be able to have {roleName} after {dueTo}.",
```

Comment:
`have the {roleName} after <t:1726931400:F>` where 1726931400 would be the epoch seconds., Discord should convert it automatically for the user.